merlin/util/duplicatebcremoval.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from scipy.spatial import cKDTree
+import networkx as nx
+import pandas as pd
```

Comment:
Can we rename this module so that in the future we could move some of the adaptive thresholding code into this file?